buildbot-host/buildmaster/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG MY_NAME="buildmaster"
-ARG MY_VERSION="v2.2.1"
-ARG IMAGE=buildbot/buildbot-master:v3.5.0
+ARG MY_VERSION="v2.2.2"
```

Comment:
Let's increase this to 2.3.0 for the buildbot update. We currently do not have a real versioning policy, but let's try to increase it somewhat sensibly anyway...

Reply:
yeah - new functionality